Wikipedia/Code/MWKArticle.m

```diff
@@ -507,4 +507,18 @@ - (NSString*)articleHTML {
     return htmlStr;
 }
 
+- (nullable NSArray<MWKTitle*>*)disambiguationTitles {
+    if (self.sections.count == 0) {
```

Comment:
if you do:

```
[self.sections firstObject] disambiguationTitles]
```

You can skip this check. `firstObject` is always safe to call.